Lib/ntpath.py

```diff
@@ -765,8 +785,8 @@ def relpath(path, start=None):
     try:
         start_abs = abspath(normpath(start))
         path_abs = abspath(normpath(path))
-        start_drive, start_rest = splitdrive(start_abs)
-        path_drive, path_rest = splitdrive(path_abs)
+        start_drive, _, start_rest = splitroot(start_abs)
+        path_drive, _, path_rest = splitroot(path_abs)
```

Comment:
This looks like a behaviour change to me (depending on whether the now-ignored root matches between `start_abs` and `path_abs`) -- I'm guessing it's intentional? Or are those corner-cases somehow already leading to the same outcome (or known-wrong currently)?

Reply:
With `splitdrive()`, the root would be at the beginning of `start_rest`. However, the root doesn't matter because the values are further processed into a list of components as follows:

```python
        start_list = [x for x in start_rest.split(sep) if x]
        path_list = [x for x in path_rest.split(sep) if x]
```

Reply:
ah, I had seen that list comprehension but misread the fact that it will also strip the separator. 👍